tailwind.config.js

```diff
@@ -13,7 +13,12 @@ module.exports = {
       {
         light: {
           ...require("daisyui/src/theming/themes")["light"],
-          primary: "#55ACD8",        }
+          primary: "#55ACD8",
```

Comment:
At the moment, this updates the daisyui's light theme. This means that we can't use custom names (ex. danger) for the colors as it'll result in an error. 

It is possible to custom theme, but I didn't want to alter the appearance of components in the UI. I could handle setting a custom theme and ensuring the `light` one is also applied in the future.